functions/event-reminders-hourly/index.js

```diff
@@ -252,10 +267,9 @@ const handler = async function (event, context) {
                   type: 'section',
                   text: {
                     type: 'mrkdwn',
-                    text: `*Announcement posted to:* <#${
-                      event.eventSlackAnnouncementsChannelId ||
-                      DEFAULT_SLACK_EVENT_CHANNEL
-                    }>`,
+                    text:
+                      `*Announcement posted to:* ` +
+                      channels.map((channel) => `<#${channel}>`).join(' '),
```

Comment:
Small tweak - let's make that `join(, )` here and earlier where we use `join(' ')`

Reply:
done